modules/ding_provider/connie_openplatform_token/connie_openplatform_token.module

```diff
@@ -72,7 +95,15 @@ function connie_openplatform_token_form() {
     '#title' => t('Open Platform token'),
     '#size' => 36,
     '#description' => t('Enter a <a href="https://github.com/DBCDK/hejmdal/blob/master/docs/oauth2.md">access token from Adgansplatformen</a> to register for the current user.'),
-    '#default_value' => connie_openplatform_token_get(),
+    '#default_value' => connie_openplatform_token_get_token(),
+  );
+
+  $form['agency'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Open Platform agency token'),
+    '#size' => 36,
+    '#description' => t('Enter a <a href="https://github.com/DBCDK/hejmdal/blob/master/docs/oauth2.md">access token from Adgansplatformen</a> to register for the current agency.'),
```

Comment:
As far as I can tell you cannot retrieve an agency access token from Adgangsplatformen. You have to do this through https://openplatform.dbc.dk/v3/.

Reply:
Updated, but I am not sure. Is it not the same behind it all.